src/BuildingRegistry.Api.Oslo/Converters/BuildingSorting.cs

```diff
@@ -0,0 +1,90 @@
+namespace BuildingRegistry.Api.Oslo.Converters
```

Comment:
Why is this class called BuildingSorting?

Reply:
Wanted to rename another file to building sorting. Guess something went wrong.